<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermainwin</class>
 <widget class="QWidget" name="usermainwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="perCabButton">
       <property name="text">
        <string> Личный кабинет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reportButton">
       <property name="text">
        <string>Сообщить об ошибке</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
